### ISSUE TITLE
Performance deterioration comparing to previous version

### ISSUE THREAD

**Issue by pipilurj** [open-mmlab/mmdetection#2466]
Hello, we have run the training script for 16 epochs using the config file "faster_rcnn_r50_fpn_1x.py", and the final MAP was 35.6 (much lower than the officially released result). And we've tried training various other models, the performances were generally not as good as what we obtained using the previous version of MMDet(we used exactly the same configs for training).
The interesting thing was that the loss during training was consistently lower than what we observed using the previous version of MMDet.
We have been using MMDetection for quite some time (appreciate a lot for the contribution), this is the first time we ran into something like this, has there been any significant modification to the code? Please give us some advice.

**Comment by hellock** [comment]
Could you tell which version (the commit id) you are using? We will see if it is reproducible.

**Comment by pipilurj** [comment]
sorry, it seems to be our own mistake, thanks for your reply!
